mne/cov.py

```diff
@@ -34,6 +34,9 @@
 from .externals.six.moves import zip
 
 
+DEFAULTS = dict(scalings=dict(eeg=1e6, grad=1e13, mag=1e15))
```

Comment:
these values are not always the best. For whitening the valus that you removed were better.